_code-samples/batch/js/multiAccountBatch.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * XRP Ledger Batch Transactions Tutorial
+ *
+ * This tutorial demonstrates how to use the Batch transaction feature (XLS-56)
+ * to perform a multi-account batch transaction.
+ * Concept doc: https://xrpl.org/docs/concepts/transactions/batch-transactions
+ * Reference doc: https://xrpl.org/docs/references/protocol/transactions/types/batch
+*/
+
+import xrpl from "xrpl"
+
+const client = new xrpl.Client("wss://s.devnet.rippletest.net:51233/")
+await client.connect()
+
+// Create and fund wallets
+console.log("=== Funding new wallets from faucet... ===")
+const { wallet: alice } = await client.fundWallet()
+const { wallet: bob } = await client.fundWallet()
+const { wallet: charlie } = await client.fundWallet()
+const { wallet: thirdPartyWallet } = await client.fundWallet()
+
+console.log(`Alice: ${alice.address}, Balance: ${await client.getXrpBalance(alice.address)} XRP`)
+console.log(`Bob: ${bob.address}, Balance: ${await client.getXrpBalance(bob.address)} XRP`)
+console.log(`Charlie: ${charlie.address}, Balance: ${await client.getXrpBalance(charlie.address)} XRP`)
+console.log(`Third-party wallet: ${thirdPartyWallet.address}, Balance: ${await client.getXrpBalance(thirdPartyWallet.address)} XRP`)
+
+// Create inner transactions  --------------------------------------------
+// REQUIRED: Inner transactions MUST have the tfInnerBatchTxn flag (0x40000000).
+// This marks them as part of a batch (allows Fee: 0 and empty SigningPubKey).
+
+// Transaction 1: Charlie pays Alice
+const charliePayment = {
+  TransactionType: "Payment",
+  Account: charlie.address,
+  Destination: alice.address,
+  Amount: xrpl.xrpToDrops(50),
+  Flags: xrpl.GlobalFlags.tfInnerBatchTxn // THIS IS REQUIRED
+}
+
+// Transaction 2: Bob pays Alice
+const bobPayment = {
+  TransactionType: "Payment",
+  Account: bob.address,
+  Destination: alice.address,
+  Amount: xrpl.xrpToDrops(50),
+  Flags: xrpl.GlobalFlags.tfInnerBatchTxn // THIS IS REQUIRED
+}
+
+// Send Batch transaction --------------------------------------------
+console.log("\n=== Creating Batch transaction... ===")
+const batchTx = {
+  TransactionType: "Batch",
+  Account: thirdPartyWallet.address,
+  Flags: xrpl.BatchFlags.tfAllOrNothing, // tfAllOrNothing: All inner transactions must succeed
+  // Must include a minimum of 2 transactions and a maximum of 8 transactions.
+  RawTransactions: [
+    { RawTransaction: charliePayment },
+    { RawTransaction: bobPayment },
+  ]
+}
+console.log(JSON.stringify(batchTx, null, 2))
+
+// Validate the transaction structure
+xrpl.validate(batchTx)
+
+// Set the expected number of signers for this transaction.
+// "autofill" will automatically add Fee: "0" and SigningPubKey: "".
```

Comment:
```suggestion
// "autofill" will automatically add Fee: "0" and SigningPubKey: "" to inner transactions.
```
It may also be worth commenting that the 2 corresponds to the number of signers in the code sample itself.